_source/handbook/01_introduction.md

```diff
@@ -1,6 +1,7 @@
 ---
-permalink: /handbook/introduction
-redirect_from: /handbook/
+title: Introduction
+permalink: /handbook/introduction.html
+nav_prefix: 01
```

Comment:
Possible to preserve the previous setup that pulled these numbers from the filename?

Reply:
Probably possible, I'll investigate.

Reply:
OK, automatic nav prefixes are in place now, via a [custom filter](https://github.com/hotwired/stimulus-site/pull/7/files#diff-404a5896c4e454dd9f90bcb2f353ecccebfc02483f2dc2befd3acc4f79f5eebd). This is one of my favorite things about 11ty. Instead of using a bunch of template logic and pre-baked Liquid filters, you can step out to Javascript land and make stuff happen.

Reply:
Nice! Looks like it borked the Reference nav… Easy to scope by section (the Handbook)? 
![Screen Shot 2021-02-03 at 1 50 08 PM](https://user-images.githubusercontent.com/5355/106794818-558d1880-6627-11eb-9121-7d6c069d3278.png)

Reply:
Woops! Fixed now. It's not scoped to the reference section specifically, it just won't output the extra characters if the prefix value is empty.